sci-biology/afni/afni-0_pre20130927.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit multilib pax-utils toolchain-funcs
+
+DESCRIPTION="An open-source environment for processing and displaying functional MRI data"
+HOMEPAGE="http://afni.nimh.nih.gov/"
+SRC_URI="http://dev.gentoo.org/~jlec/distfiles/${P}.tgz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+RDEPEND="x11-libs/motif[-static-libs]"
+
+# x11-libs/motif[static-libs] breaks the build.
+# See upstream discussion
+# http://afni.nimh.nih.gov/afni/community/board/read.php?1,85348,85348#msg-85348
+
+DEPEND="${RDEPEND}
+	app-shells/tcsh
+	sci-libs/gsl
+	dev-libs/expat
+	x11-libs/libXpm
+	media-libs/netpbm
+	media-video/mpeg-tools
+	x11-libs/libGLw
+	media-libs/qhull
+	sys-devel/llvm
+	media-video/mpeg-tools
+	media-libs/libjpeg-turbo"
```

Comment:
You currently don't need most of these at build time because afni has a bundled copy - which is usually bad(TM). A prime example is they have a copy of jpeg6b. So not only will you have to put most of these in `RDEPEND` it will have to be unbundled.

To go back to jlec's point it is fairly uncommon for a library to be needed at run time but not at build time for software built from source. Exercise to the reader: describe a scenario in which it can happen.